.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,37 @@
+name: Check build pass/fail
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+
+    # Setup the flutter web build
+    - uses: subosito/flutter-action@v1.2.0
+      with:
+        channel: beta
+      if: ${{ runner.os == "Windows" }}
```

Comment:
The if and the run should be on the same indent level
like this:
```yaml
- if: runner.os == "Windows"
  run: flutter config --enable-windows-desktop
```
Also note you can drop the `${{ }}` in if statements